libs/tup-components/src/news/UserNews.tsx

```diff
@@ -7,6 +7,8 @@ import {
 } from '@tacc/core-components';
 import styles from './UserNews.module.css';
 
+const NEWS_DASHBOARD_DISPLAY_LIMIT = 3;
```

Comment:
Moved limit to a constant[^1] and lowered it.

[^1]: Just like in `libs/tup-components/src/tickets/TicketsTable.tsx`.